cmd/util/cmd/common/environment.go

```diff
@@ -0,0 +1,102 @@
+package common
+
+import (
+	"fmt"
+
+	"github.com/onflow/flow-core-contracts/lib/go/templates"
+)
+
+var (
+	getInfoForProposedNodesScriptLocalnet = []byte(`
+		import FlowIDTableStaking from 0xf8d6e0586b0a20c7
+		pub fun main(): [FlowIDTableStaking.NodeInfo] {
+			let nodeIDs = FlowIDTableStaking.getProposedNodeIDs()
+		
+			var infos: [FlowIDTableStaking.NodeInfo] = []
+			for nodeID in nodeIDs {
+				let node = FlowIDTableStaking.NodeInfo(nodeID: nodeID)
+				infos.append(node)
+			}
+		
+			return infos
+	}`)
+
+	getInfoForProposedNodesScriptTestnet = []byte(`
+		import FlowIDTableStaking from 0x9eca2b38b18b5dfe
+		pub fun main(): [FlowIDTableStaking.NodeInfo] {
+			let nodeIDs = FlowIDTableStaking.getProposedNodeIDs()
+		
+			var infos: [FlowIDTableStaking.NodeInfo] = []
+			for nodeID in nodeIDs {
+				let node = FlowIDTableStaking.NodeInfo(nodeID: nodeID)
+				infos.append(node)
+			}
+		
+			return infos
+	}`)
+
+	getInfoForProposedNodesScriptMainnet = []byte(`
+		import FlowIDTableStaking from 0x8624b52f9ddcd04a
+		pub fun main(): [FlowIDTableStaking.NodeInfo] {
+			let nodeIDs = FlowIDTableStaking.getProposedNodeIDs()
+		
+			var infos: [FlowIDTableStaking.NodeInfo] = []
+			for nodeID in nodeIDs {
+				let node = FlowIDTableStaking.NodeInfo(nodeID: nodeID)
+				infos.append(node)
+			}
+		
+			return infos
+	}`)
+)
+
+func GetNodeInfoForProposedNodesScript(network string) ([]byte, error) {
+	if network == "mainnet" {
+		return getInfoForProposedNodesScriptMainnet, nil
+	}
+
+	if network == "testnet" {
+		return getInfoForProposedNodesScriptTestnet, nil
+	}
+
+	if network == "localnet" {
+		return getInfoForProposedNodesScriptLocalnet, nil
+	}
+
+	return nil, fmt.Errorf("invalid network string expecting one of ( mainnet | testnet | localnet )")
+}
+
+func EnvFromNetwork(network string) (templates.Environment, error) {
```

Comment:
This is unused now I believe - can remove

Reply:
https://github.com/onflow/flow-go/pull/1988/commits/93b09c2b5b78314c38c0cb496c64378b5e740376